<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset theme="elf-dissector">
    <normaloff/>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="SidePane" name="viewSelector">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="ElfStructureView" name="elfStructureView"/>
      <widget class="SizeTreeMapView" name="sizeTreeMapView"/>
      <widget class="DependencyView" name="dependencyView"/>
      <widget class="TypeView" name="typeView"/>
      <widget class="LoadBenchmarkView" name="loadTimeView"/>
      <widget class="IssuesView" name="issuesView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;Actions</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionReopenPreviousFile"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionReopenPreviousFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="document-open-recent">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Reopen &amp;Previously Loaded Files</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SizeTreeMapView</class>
   <extends>QWidget</extends>
   <header>views/sizetreemapview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DependencyView</class>
   <extends>QWidget</extends>
   <header>views/dependencyview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ElfStructureView</class>
   <extends>QWidget</extends>
   <header>views/elfstructureview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LoadBenchmarkView</class>
   <extends>QWidget</extends>
   <header>views/loadbenchmarkview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TypeView</class>
   <extends>QWidget</extends>
   <header>views/typeview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IssuesView</class>
   <extends>QWidget</extends>
   <header>views/issuesview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SidePane</class>
   <extends>QListView</extends>
   <header>widgets/sidepane.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
